includes/Core/Modules/Module.php

```diff
@@ -338,6 +338,9 @@ final protected function execute_data_request( Data_Request $data ) {
 			$datapoint    = $this->get_datapoint_definition( "{$data->method}:{$data->datapoint}" );
 			$oauth_client = $this->get_oauth_client_for_datapoint( $datapoint );
 
+			// Set request as using shared credentials to false.
```

Comment:
Please use comments to add context and explain why we're doing something that may otherwise not make sense. This comment is redundant as-is because it states what it does rather than why, which would be good to call out here especially with the default value and `finally` below.
```suggestion
			// Always reset this property first to ensure it is only set true for the current request.
```